05_stringer/CameronSchafer/main_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "127.0.0.1\n"
+	actual := buf.String()
+
+	assert.Equalf(t, expected, actual,
+		"Output buffer does not match with expected output.")
+}
+
+func TestFmtStringer(t *testing.T) {
+	testCases := map[string]struct {
+		input       IPAddr
+		expectedArr string
+	}{
+		"127.0.0.1": {input: IPAddr{127, 0, 0, 1},
+			expectedArr: "127.0.0.1"},
+		"192.168.1.1": {input: IPAddr{192, 168, 1, 1},
```

Comment:
I'm not really sure what these two tests are testing `192.168.1.1` and `204.180.10.255`. Just remember to select your tests with a purpose, and look for the possible edge cases and guard around them.

Might have been worthwhile to put a test around the max byte value `255.255.255.255` to ensure that a byte array was used as the underlying type.